src/tasks/dump_db/dump-import.sql.hbs

```diff
@@ -1,3 +1,10 @@
+BEGIN;
+    -- Disable triggers on each table.
+{{~#each tables}}
+    ALTER TABLE "{{this.name}}" DISABLE TRIGGER ALL;
+{{~/each}}
+COMMIT;
```

Comment:
Why is this wrapped in a separate transaction? Can't we do this as part of the "main" transaction, or do we need to commit to make sure the triggers are actually disabled?

Reply:
You are correct that the extra transactions are not needed. To answer your question _Why …?_: I was struggling to get the `updated_at` to survive through an import and didn't want my not knowing very much about how triggers are handled within the database to get in the way of my attempts. Once you asked about doing it in a single transaction, I tried again. Worked great!

I've added a commit pulling out the extra COMMITs and BEGINs.

Reply:
Thanks for the update!

The advantage of doing everything in a single transaction is that everything will be rolled back at once in case of an error. It doesn't matter too much here, since this is generally performed on a clean database, but it still makes more sense to me that the inport either succeeds atomically, or has no effect at all.